docs/integrations/destinations/pinecone.md

```diff
@@ -68,12 +68,13 @@ For testing purposes, it's also possible to use the [Fake embeddings](https://py
 
 To get started, use the [Pinecone web UI or API](https://docs.pinecone.io/docs/quickstart) to create a project and an index before running the destination. All streams will be indexed into the same index, the `_ab_stream` metadata field is used to distinguish between streams. Overall, the size of the metadata fields is limited to 30KB per document. 
 
-OpenAI and Fake embeddings produce vectors with 1536 dimensions, and the Cohere embeddings produce vectors with 1024 dimensions. Make sure to configure the index accordingly.
+OpenAI and Fake embeddings produce vectors with 1536 dimensions, and the Cohere embeddings produce vectors with 1024 dimensions. Make sure to configure the index accordi../../../docs/integrations/destinations/pinecone.mdngly.
```

Comment:
What happened here? I don't think this path was intended to be added?

Reply:
Oops